docs/interfaces.md

```diff
@@ -357,11 +357,16 @@ Can use kernel tracing facilities. This is restricted because it gives
 privileged access to all processes on the system and should only be used with
 trusted apps.
 
-Usage: reserved
```

Comment:
https://github.com/snapcore/snapd/pull/1502#discussion_r74417508

System-trace happens to be in old format too. Fix it in the same commit with tpm.